esdb-client-spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -6,10 +7,12 @@ dependencies {
     compileOnly("org.springframework.boot:spring-boot-starter-validation")
     compileOnly("org.springframework.boot:spring-boot-autoconfigure")
     compileOnly("org.springframework.boot:spring-boot-starter-actuator")
+    compileOnly("io.opentelemetry.instrumentation:opentelemetry-spring-boot-starter")
```

Comment:
those 2 added dependencies are exactly the opposite as defined in esdb-client/build.gradle.kts with respect to their scopes: why?